test/e2e/page-objects/common.ts

```diff
@@ -2,3 +2,9 @@ export type RawLocator =
   | string
   | { css?: string; text?: string }
   | { tag: string; text: string };
+
+export enum ACCOUNT_TYPE {
+  ETHEREUM,
+  BITCOIN,
+  SOLANA,
+}
```

Comment:
IMO, this should be in `test/e2e/flask/helpers.js`

Also, this does not follow our naming convention, so it should be:

```suggestion
export enum AccountType {
  Ethereum,
  Bitcoin,
  Solana,
}
```

Reply:
Oh sorry, I misread the file name actually.. I did not see the `common.ts` part 🙈

We can probably keep it here for now indeed 👍